ocs_ci/framework/conf/default_config.yaml

```diff
@@ -36,6 +36,8 @@ RUN:
   skip_reason_test_found: {}
   skipped_tests_ceph_health: 0
   number_of_tests: None
+  skipped_on_ceph_health_ratio: 0
+  skipped_on_ceph_health_threshold: 0.25
```

Comment:
Please add the new parameters to the [conf/README.md](https://github.com/red-hat-storage/ocs-ci/blob/master/conf/README.md) (I miss that in the previous PR).

Reply:
@ebenahar have you seen my input in the mail thread?  I think that 25% is too high. What about to lover to 10 or even 5%